### PR TITLE
Add ssh keys post

### PR DESCRIPTION
**Pull request by MacTynow**
First post of the blog.

### CODE REVIEW

File: _posts/2016-09-08-ssh-keys-github.markdown [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+categories: devops aws
+---
+Recently we decided to improve the way we manage ssh keys on our servers. We're running our stack on Elastic Beanstalk and the way we would push our ssh keys would be to use a config file appending the user's key to the server's `authorized_keys` file. This solution is not really optimal because it means we're storing the public keys in the repository, access control is quite low and you'd have to go through a whole pull request process just to be able to ssh to a machine.
+
+# Goals
+1. Ideally, we'd like to be able to organise users in groups, each groups having access to different servers. 
+2. Teams would manage who has access themselves. 
+3. Revoking/Granting access should be immediate. 
+4. Public keys wouldn't be stored in a repository.
+5. Users should be able to access the server from different machines.
```
Comment (lenguyenthedat) [inline _posts/2016-09-08-ssh-keys-github.markdown]: `servers`

File: _posts/2016-09-08-ssh-keys-github.markdown [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+# Security 
+On the security side, the keys being public, there are only a few ways we can be attacked : 
+- Github being spoofed and serving public keys that are matching the attacker's private key - unlikely
+- Our github account being hacked and a malicious user being added to the teams in order to get access 
+
+# Fallback 
+In case of failure of the process, sshd falls back by default to the `authorized_keys` file, where we still have one key stored.
+
+[better-ssh]: https://gist.github.com/sivel/c68f601137ef9063efd7
```
Comment (lenguyenthedat) [inline _posts/2016-09-08-ssh-keys-github.markdown]: Shall we fork it just in case this link get deleted?

Comment (MacTynow) [inline _posts/2016-09-08-ssh-keys-github.markdown]: Good idea, I did. Will replace the link if it becomes dead.

File: _config.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@ email: tech YOU KNOW honestbee.com
 description: > # this means to ignore newlines until "baseurl:"
   This is the tech blog for the engineering teams at Honestbee.
```
Comment (lenguyenthedat) [inline _config.yml]: s/Honestbee/honestbee

**Pull request merged**